src/vm/excep.cpp

```diff
@@ -7233,8 +7233,9 @@ EXTERN_C void JIT_WriteBarrier_Debug_End();
 EXTERN_C void FCallMemcpy_End();
 #endif
 
-static
-bool IsIPExcluded(UINT_PTR uControlPc)
+// Check if the passed in instruction pointer is in one of the
+// JIT helper functions.
+bool IsIPInJitHelper(UINT_PTR uControlPc)
```

Comment:
Although the `IsIPInMarkedJitHelper` sounds clearer to me in the following test. With the name you have suggested:
`if (ExecutionManager::IsManagedCode(controlPc) || ShouldHandleJitHelperFault(controlPc))`
@jkotas Looking at such code would make me think me that the controlPc points to a jit helper and the function is meant to determine whether to handle it or not, while the real purpose of the function is to check whether the ip is inside of a subset of the jitter helpers.
So I'd still prefer the `IsIPInMarkedJitHelper` name I've suggested. Would it be acceptable for you?